src/lib/utils/thorchain/hooks/useSendThorTx.tsx

```diff
@@ -408,44 +399,6 @@ export const useSendThorTx = ({
       }
     })()
 
-    const maybeSafeTx = await fetchSafeTransactionInfo({
-      safeTxHash: _txId,
-      fetchIsSmartContractAddressQuery,
-      address: fromAccountId(accountId).account,
-      chainId: fromAccountId(accountId).chainId,
-    })
-
-    const _txIdLink = getTxLink({
-      defaultExplorerBaseUrl: 'https://viewblock.io/thorchain/tx/',
-      txId: _txId ?? '',
-      stepSource: SwapperName.Thorchain,
-      maybeSafeTx,
-      address: fromAccountId(accountId).account,
-      chainId: fromAccountId(accountId).chainId,
-    })
-
-    // Only toast "Transaction sent" for non-SAFE Tx hashes - in the case of SAFE Txs, dis not a final on-chain Tx just yet
-
-    // Also note confirmed TCY un/stakes (more to come) are currently handled by the action center, with new style bottom-right
-    // toasts, so we don't want to toast the Tx broadcasted one here, or the two diff toasts would look very odd to the user
-    // Toasts on Tx broadcasted are quite annoying for such a fast blocktime anyway, so they should probably all go away eventually anyway
-    if (!maybeSafeTx?.isSafeTxHash && !['stakeTcy', 'unstakeTcy'].includes(action)) {
```

Comment:
Since we needed one more monkey patch here and the goal is to go away from those "old" notifications, figured out we might as well, this should now be considered spew